### ISSUE TITLE
Set up a dedicated domain

### ISSUE THREAD

**Issue by dslmeinte** [ehn-dcc-development/eu-dcc-site#7]
Something like `eu-dcc-site.eu`?

**Comment by skounis** [comment]
Why not just dcc.eu:
1. eu-dcc-site.eu: it's a site, so do we need it in the name? 
2. eu-dcc.eu: prefix and suffix (tld) overlap

https://dcc.eu

**Comment by dslmeinte** [comment]
Agreed: `dcc.eu` is much better. Problem: it apparently costs 5K€ excl. VAT 😢

**Comment by skounis** [comment]
Just saw it.

**Comment by skounis** [comment]
EU-DCC.org is available

**Comment by dslmeinte** [comment]
Under which organization/legal entity could we set this domain up?
